Modules/Relationships/src/Relationships.Infrastructure/Extensions/RelationshipChangeQueryableExtensions.cs

```diff
@@ -7,18 +7,18 @@ namespace Backbone.Modules.Relationships.Infrastructure.Extensions;
 
 public static class RelationshipChangeQueryableExtensions
 {
-    public static IQueryable<RelationshipChange> CompletedAt(this IQueryable<RelationshipChange> query, OptionalDateRange completedAt)
+    public static IQueryable<RelationshipChange> CompletedAt(this IQueryable<RelationshipChange> query, OptionalDateRange? completedAt)
     {
         var newQuery = query;
 
         if (completedAt == null)
             return newQuery;
 
         if (completedAt.From != default)
-            newQuery = newQuery.Where(r => r.Response.CreatedAt >= completedAt.From);
+            newQuery = newQuery.Where(r => r.Response != null && r.Response.CreatedAt >= completedAt.From);
```

Comment:
I don't quite understand this null check - it implies that elements in the `IQueryable<RelationshipChange> query` collection could be null... @tnotheis could they?

There is a similar situation in `Modules/Relationships/src/Relationships.Infrastructure/Extensions/RelationshipQueryableExtensions.cs` and there you have not introduced any such checks.

More so, ReSharper doesn't complain when I remove it.

Reply:
Response prop is nullable, hence there is the ReSharper warning about it.

Reply:
I agree. Still, my ReSharper doesn't complain when I remove the `null` check, maybe Timo can take a look and see what's going on.